### PR TITLE
Create demo

### PR DESCRIPTION
**Pull request by wirednkod**
This isa demo that contains Multiple relay chains and a parachain. 
The page connect to Polkadot, Kusama, Westend and Rococo and display the latest header (as previously multinetwork demo was). In addition it tries to connect to Parachain Westmint and display some data from the Parachain (as previously parachain-demo was).
The scope of this demo is for running manually connection attemps to relay and parachains when changes take place.

### CODE REVIEW

File: projects/demo/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    "typecheck:watch": "tsc -b --watch",
+    "copy-assets": "cp -R ./src/assets ./dist",
+    "build": "yarn clean && yarn copy-assets && yarn typecheck && webpack",
+    "dev": "yarn clean && yarn copy-assets && concurrently \"yarn typecheck:watch\" \"webpack serve --config webpack.config.js\"",
+    "test": "exit 0; #No tests",
+    "lint": "yarn eslint . --ext .ts,.tsx"
+  },
+  "dependencies": {
+    "@polkadot/api": "^7.8.1",
+    "@substrate/connect": "^0.5.0"
```
Comment (tomaka) [inline projects/demo/package.json:25]:
```suggestion
    "@substrate/connect": "^0.6.1"
```

File: projects/burnr/src/hooks/api/useApiCreate.ts [hunk 2]
```diff
@@ -20,7 +20,8 @@ export const useApiCreate = (): ApiPromise => {
         const provider = await scClient.addWellKnownChain(
           endpoint as WellKnownChain,
         )
-        const api = await ApiPromise.create({ provider })
+        const api = new ApiPromise({ provider })
+        // const api = await ApiPromise.create({ provider })
```
Comment (tomaka) [inline projects/burnr/src/hooks/api/useApiCreate.ts:24]: ?

File: projects/demo/package.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    "typecheck": "tsc -b",
+    "typecheck:watch": "tsc -b --watch",
+    "copy-assets": "cp -R ./src/assets ./dist",
+    "build": "yarn clean && yarn copy-assets && yarn typecheck && webpack",
+    "dev": "yarn clean && yarn copy-assets && concurrently \"yarn typecheck:watch\" \"webpack serve --config webpack.config.js\"",
+    "test": "exit 0; #No tests",
+    "lint": "yarn eslint . --ext .ts,.tsx"
+  },
+  "dependencies": {
+    "@polkadot/api": "^7.8.1",
```
Comment (tomaka) [inline projects/demo/package.json:24]:
```suggestion
    "@polkadot/api": "^7.10.1",
```

File: projects/burnr/src/hooks/api/useApiCreate.ts (hunk 2 above)
Comment (wirednkod) [reply projects/burnr/src/hooks/api/useApiCreate.ts:24]: This is done erroneously 🙏🏽 thank you

**Review by tomaka — APPROVED** [review]

**Pull request merged**